### PR TITLE
fix(quantic): minor issues fixed in the Quantic Generated answer component

### PR DESCRIPTION
**Pull request by mmitiche**
- [SFINT-5297](https://coveord.atlassian.net/browse/SFINT-5297): alignment of citations adjusted.
- [SFINT-5294](https://coveord.atlassian.net/browse/SFINT-5294): label of citations updated.
- [SVCC-3278](https://coveord.atlassian.net/browse/SVCC-3276): allowing opening the feedback modal after rephrasing the generated answer.
- [SFINT-5290](https://coveord.atlassian.net/browse/SFINT-5290): adjusted font size of generated answer.
- [SFINT-5297](https://coveord.atlassian.net/browse/SFINT-5275): updating the icon of the summary rephrase button.

#### Demo:
https://github.com/coveo/ui-kit/assets/86681870/bd3dac3e-3d70-48fb-83a4-5e0bc8c99102

[SFINT-5294]: https://coveord.atlassian.net/browse/SFINT-5294?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SFINT-5290]: https://coveord.atlassian.net/browse/SFINT-5290?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[SFINT-5297]: https://coveord.atlassian.net/browse/SFINT-5297?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by erocheleau — APPROVED** [review]

File: packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.css [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 .generated-answer__answer {
   white-space: pre-wrap;
+  line-height: 1.5rem;
+  font-size: 15px;
```
[the remark this answers is not in the document]
Comment (SimonMilord) [reply packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.css:10]: `font-size: 0.9375rem;` 😅

[the remark this answers is not in the document]
Comment (SimonMilord) [reply packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.css:10]: or 0.9375rem?

**Review by SimonMilord — APPROVED** [review]
👍

File: packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (mmitiche) [reply packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.css:10]: Unfortunately no there are no public slds variable for such value.

**Pull request merged**